fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1,299 @@
+
```

Comment:
But that's what is happening now? The issue is just that this is an additional copy of the `.rubocop.yml` inside the repo.